flexget/plugins/notifiers/notify.py

```diff
@@ -156,6 +162,53 @@ def on_task_start(self, task, config):
     on_task_exit = send_notification
 
 
+class NotifyEntries(NotifyBase):
+    def on_task_exit(self, task, config):
+        config['scope'] = 'entries'
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, config)
+
+
+class NotifyTask(NotifyBase):
+    def on_task_exit(self, task, config):
+        config['scope'] = 'task'
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, config)
+
+
+class NotifyAbort(NotifyBase):
+    def on_task_abort(self, task, config):
+        if task.silent_abort:
+            return
+
+        title = 'Task {{ task_name }} has aborted!'
+        message = 'Reason: {{ task.abort_reason }}'
+        notify_config = {'to': config['to'],
+                         'scope': 'task',
+                         'title': title,
+                         'message': message}
+        log.debug('sending abort notification')
+        plugin.get_plugin_by_name('notify').instance.send_notification(task, notify_config)
+
+
+class NotifyCrash(NotifyBase):
+    def on_task_abort(self, task, config):
```

Comment:
you can use it in a global template. I think it feels better than a root level, since it has no merit or functionality on its own.